### ISSUE TITLE
A solution to run none-container command on Kubernetes

### ISSUE THREAD

**Issue by laoshanxi** [kvaps/kubectl-node-shell#26]
Hi,

Here is a simple solution to forward container command to outside and run on host OS, so with this you can run real native command by Kubernetes:

https://app-mesh.readthedocs.io/en/latest/success/kubernetes_run_native_application.html

**Comment by madeddie** [comment]
Interesting solution, but it requires installing appmesh on your cluster. kubectl-node-shell needs nothing else and leaves no trace

**Comment by laoshanxi** [comment]
> Interesting solution, but it requires installing appmesh on your cluster. kubectl-node-shell needs nothing else and leaves no trace

Yes, by default Kubernetes can only support container application, if you want to use Kubernetes manage native application (none-container), this can be a solution.
